packages/orbit-components/src/Pagination/README.md

```diff
@@ -20,8 +20,8 @@ Table below contains all types of the props available in the Pagination componen
 | :--------------- | :--------------- | :--------- | :----------------------------------------------------------------------------------------- |
 | dataTest         | `string`         |            | Optional prop for testing purposes.                                                        |
 | hideLabels       | `boolean`        | `true`     | If `false`, the Previous and Next labels will be visible.                                  |
-| labelPrev        | `React.Node`     |            | The text label for the previous page call to action.                                       |
-| labelNext        | `React.Node`     |            | The text label for the next page call to action.                                           |
+| labelPrev        | `string`         |            | The text label for the previous page call to action.                                       |
+| labelNext        | `string`         |            | The text label for the next page call to action.                                           |
 | labelProgress    | `React.Node`     |            | The text label for progress indicator.                                                     |
```

Comment:
What about this one? Looks it might need the same treatment. Though I'm not sure how you use it under the hood.